### DISCUSSION TITLE
The "Within" in the search "Near Location" is showing only a maximum of 25 kilometers

### DISCUSSION THREAD

**Discussion by adminaa87** [code4recovery/12-step-meeting-list#840] [Q&A] [answered]
Hello,

The "Within..." in the search"Near Location" is showing only a maximum of 25 kilometers.
In some areas, the distance is more than 25 kilometers, so this search from a postal code is not showing any result. 

We have TSML Version 3.14.4.

How can I add a bigger "Within" in the drop down list?

Thank you.

Benoit

**Comment by tech2serve** [comment, marked as answer]
Hi Benoit,

Please check [this entry](https://github.com/code4recovery/12-step-meeting-list/wiki/Frequently-Asked-Questions#how-do-i-change-the-default-search-radius-for-location-searches) in our FAQ. 

Tim

**Comment by adminaa87** [comment]
Hello,

Thank you. It's working fine ;-)

Another question about that: I had to reinstall a new site TSML and it is showing distance in Miles instead of kilometers. 

Any idea ?

**Reply by joshreisner** [reply]
there's a setting in Meetings > Import & Settings > Distance Units

<img width="579" alt="Screen Shot 2022-08-24 at 1 52 55 PM" src="https://user-images.githubusercontent.com/1551689/186521298-89b0cc5c-6aeb-477a-a585-a4aa36815a74.png">

**Comment by adminaa87** [comment]
Hello, 
We installed the last version o TSML. 
I dont remembre how to add more radius. I already ask the question, but this article is not available:
https://github.com/code4recovery/12-step-meeting-list/wiki/Frequently-Asked-Questions#how-do-i-change-the-default-search-radius-for-location-searches

Another question: with this new version, we cannot query based on postal code near me. This is relay useful since we have a form that the visitor enter is postal code and we redirect to a TSML link with parameter search near that postal code. 18000 visitors have used that form.

thank you 

Benoît

**Reply by joshreisner** [reply]
the link above moved to https://wordpress.org/plugins/12-step-meeting-list/#how%20do%20i%20change%20the%20default%20search%20radius%20for%20location%20searches%3F

we haven't removed any features, so i'm guessing something on your site has changed. could you provide a link to the form that is no longer working?

**Comment by adminaa87** [comment]
Hello,

I am sending you a PDF with more informations.

Thank you

Benoît



De : Josh Reisner ***@***.***>
Envoyé : 1 mai 2024 00:07
À : code4recovery/12-step-meeting-list ***@***.***>
Cc : Admin systèmes ***@***.***>; Author ***@***.***>
Objet : Re: [code4recovery/12-step-meeting-list] The "Within" in the search "Near Location" is showing only a maximum of 25 kilometers (Discussion #840)


the link above moved to https://wordpress.org/plugins/12-step-meeting-list/#how%20do%20i%20change%20the%20default%20search%20radius%20for%20location%20searches%3F

we haven't removed any features, so i'm guessing something on your site has changed. could you provide a link to the form that is no longer working?

**Reply by joshreisner** [reply]
ok just fyi email attachments don't work on github, youll need to come to the discussion and upload it

**Comment by adminaa87** [comment]
I am sending you a PDF with more informations.
[2024-05-01 problem TSML TSML UI mode.pdf](https://github.com/code4recovery/12-step-meeting-list/files/15181777/2024-05-01.problem.TSML.TSML.UI.mode.pdf)

**Reply by joshreisner** [reply]
ok, we can add 50 & 100 distance options to TSML UI. i've created a ticket: https://github.com/code4recovery/tsml-ui/issues/382

i'm not quite sure what you mean by "And the title label of column are in english." do you mean they now say `1 km` instead of `Dans 1 Kilometre`?

that's correct the query string has changed. use `search` now instead of `tsml-query`.  i see it's not adding that to the URL and have created a ticket https://github.com/code4recovery/tsml-ui/issues/383 but you can make that change to your form now and it should work

**Comment by adminaa87** [comment]
_Josh says: _i'm not quite sure what you mean by "And the title label of column are in english._

There are differences between Legacy UI and TSML UI about labels language. 


[2024-05-03 french english mixed between Legacy and TSML UI .pdf](https://github.com/code4recovery/12-step-meeting-list/files/15200912/2024-05-03.french.english.mixed.between.Legacy.and.TSML.UI.pdf)

**Reply by joshreisner** [reply]
thanks for the clarification. created an issue: https://github.com/code4recovery/12-step-meeting-list/issues/1414
